src/apps/activity_assignments/crud/assignments.py

```diff
@@ -102,9 +225,18 @@ async def get_by_applet_and_respondent(
         query = (
             select(ActivityAssigmentSchema)
             .outerjoin(ActivitySchema, ActivitySchema.id == ActivityAssigmentSchema.activity_id)
-            .outerjoin(ActivityFlowSchema, ActivityFlowSchema.id == ActivityAssigmentSchema.activity_flow_id)
-            .join(respondent_schema, respondent_schema.id == ActivityAssigmentSchema.respondent_subject_id)
-            .join(target_schema, target_schema.id == ActivityAssigmentSchema.target_subject_id)
+            .outerjoin(
+                ActivityFlowSchema,
+                ActivityFlowSchema.id == ActivityAssigmentSchema.activity_flow_id,
+            )
```

Comment:
please, try to not change formatting of code, only if it really needed, again added comma at end

Reply:
applied changes and formatting using ruff with 120 characters limit